webapp/public/index.html

```diff
@@ -21,22 +21,28 @@
       name="description"
       content="The dashboard is an open source project for monitoring network data and block producer infrastructure for EOSIO blockchains."
```

Comment:
EOSIO dashboard is a network monitor featuring real-time data on block producer nodes and blockchain infrastructure for multiple EOSIO chains.